benchmarks/allocator_benchmarks.cpp

```diff
@@ -50,11 +49,6 @@ class allocatorBenchmark : public ::benchmark::Fixture {
     }
   }
 
-  void coalesceBench(benchmark::State &st) {
```

Comment:
I think I am okay with the removal of this benchmark?  What was the reason?

Reply:
Coalesce is no longer accessible.

Reply:
Not directly